documentation/src/main/asciidoc/topics/ref_performance_tuning.adoc

```diff
@@ -79,6 +79,95 @@ The recommended values are:
 * send window size - 640 KB
 * receive window size - 25 MB
 
+== Thread pool settings
+
+A thread pool manages threads by assigning them concurrent tasks.
+Additionally, a thread pool can improve system performance by using any idle thread instead of creating threads.
+
+Thread pools limit the creation and deletion of threads to active worker threads, so that the amount of threads does not typically exceed the amount of tasks.
+This helps reduce the memory footprint of your {brandname} server.
+
+You can configure your `threading` subsystem, so that you can better manage thread pools and their resources. 
+
+The following table outlines common `threading` subsystem components that you can configure:
+
+[cols="1,1,3"]
+|===
+| Component  | Attribute type |Description
+
+|`blocking-bounded-queue-thread-pool`|string and integer | A thread pool executor that can run blocking operations. 
+Executor checks that the number of running threads does not exceed the core size; otherwise, the executor creates new threads. 
+The thread pool uses a best effort approach to maintain a low number of running threads, such as reusing idle threads where possible.
+This behavior differs from the JRE's `ThreadPoolExecutor` that creates new threads for each generated request until core size limitations apply.
+Attributes include,`core-threads`, `keepalive-time`, `queue-length`, `thread-factory`, `max-threads`, and `name`.
+
+|`cached-thread-pool` |string| A thread pool executor that creates new threads when an idle thread cannot be used with no bounds. The `name` and `thread-factory` attributes are mandatory.
+
+|`non-blocking-bounded-queue-thread-pool` | string and integer a| A thread pool executor that can run non-blocking operations. 
+Executor creates new threads when the number of running threads is less that the core size. 
+Idle threads are placed in a queue.
+
+Includes similar attributes to the blocking-bounded-queue-thread-pool` component.
+
+|`scheduled-thread-pool`|string|A thread pool executor that creates a single-threaded executor that schedules commands to run at a specific time.
+The `name` and `thread-factory` attributes are mandatory. 
+
+|`thread-factory` |string| Specifies metadata for your threads pools. 
+The `name` attribute and the `thread-name-pattern` attributes are mandatory. 
+Optional attributes include `group-name` and `priority`.
+|===
+
+Your {brandname} configuration contains a cache container security configuration that applies default settings to thread pools.
+You can change these settings to meet your needs.
```

Comment:
The user cannot change the defaults as this seems to imply. They can only define their own explicit thread pool as outlined in the previous sections.

Reply:
Removed this sentence. Thanks!